rasa/core/training/interactive.py

```diff
@@ -78,6 +78,8 @@
     "domain": "domain.yml",
 }
 
+SAVE_IN_E2E = {"value": False}
```

Comment:
is there any reason this needs to be a dict and can't just be a boolean?

Reply:
Actually i need a global variable, and dict works fine but boolean doesn't in my case :(

Reply:
do you know why it doesn't work? (or can you tell me more about what you need it for?)

Reply:
i am using server_args to pass e2e, and updating it here inside `run_interactive_learning` function, if i do something like `SAVE_IN_E2E = True` it doesn't affect the local variable, so i need a dictionary
https://github.com/chidauri/rasa/blob/d8b2fe1fdd857dc1b2b233c30b3d3ad99a25faca/rasa/core/training/interactive.py#L1556

Reply:
ah right, you need to declare it to be global:

at the beginning of the `run_interactive_learning` function, just put `global SAVE_IN_E2E `